revad/Chart.yaml

```diff
@@ -23,6 +23,7 @@ keywords:
   - sync-and-share
 annotations:
   artifacthub.io/changes: |
+    - "Add imagePullSecrets to deployment"
     - "Update revad image to v1.18.0"
```

Comment:
```suggestion
```

You can remove this line from the changelog as it was already released.

Reply:
fixed by 5e1a2568bf52ffaa97b46f1f91141f7616da6671